gen3/bin/gitops.sh

```diff
@@ -651,6 +737,12 @@ if [[ -z "$GEN3_SOURCE_ONLY" ]]; then
     "dotag")
       gen3_gitops_repo_dotag "$@"
       ;;
+    "tfplan-vpc")
+      gen3_run_tfplan_vpc "$@"
+      ;;
+    "tfplan-eks")
+      gen3_run_tfplan_eks "$@"
+      ;;
     *)
```

Comment:
Maybe structure this, so we have a `tfplan` sub-command, that takes which plan to run as an argument - ex:
```
    "tfplan")
        gen3_run_tfplan "$@"
```